.github/workflows/build-firmware.yaml

```diff
@@ -194,9 +194,9 @@ jobs:
 
     steps:
     - name: Execution throttle early exit
-      if: ${{ matrix.skip-rate }}
+      if: ${{ matrix.skip-rate && !contains(github.ref, 'release') }}
```

Comment:
Please add code comments explaining what it does and how? i.e. both user documentation and implementation details?